randovania/games/fusion/logic_database/Sector 4 AQA.txt

```diff
@@ -1271,11 +1305,15 @@ Extra - room_id: [33]
   > Pickup (Missile Tank)
       Trivial
   > Door to Pump Control Access
-      Any of the following:
-          Can Use Springball
-          All of the following:
-              Can Use Bombs
-              Gravity Suit or After Sector 4 Water Level Lowered
+      All of the following:
+          Any of the following:
```

Comment:
isnt a mid air morph possible here too? looks the same hight as in the cubby hole

Reply:
you need at least one block above

Reply:
oh then cubby hole was higher. my b